arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -499,7 +499,6 @@
           {
             "name": "parameters",
             "in": "body",
-            "required": true,
```

Comment:
@amarzavery and @johanste, based on my reading of that link, an empty body is valid, meaning nothing needs to be passed in the request body (content-length: 0). This would be achieved by removing the required field for the parameters, since it will still have a content-type as well as a content-length.